src/tequila/circuit/visualize.py

```diff
@@ -0,0 +1,156 @@
+import tequila
+from typing import Optional, List
+import matplotlib.pyplot as plt
+import matplotlib.figure as figure
+import networkx
```

Comment:
see comment above.

You can also handle it similar to circuit.py in the function "to_networkx"

Reply:
best solution: check if the module exists and if not throw a TequilaException in the corresponding function